templates/registry/markdown/body_field_table.j2

```diff
@@ -0,0 +1,15 @@
+{% import 'requirement.j2' as requirement %}
+{% import 'stability.j2' as stability %}
+{% import 'notes.j2' as notes %}
+{% import 'enum_macros.j2' as enums %}
+{% import 'examples_macros.j2' as examples %}
+{% macro flatten(fields, ns, depth) %}{% if fields %}{% for f in fields | sort(attribute="id") %}
+{% set ns.flat = [ns.flat, [{'field':f,'depth':depth}]] | flatten %}{% if f.fields %}{% set _= flatten(f.fields, ns, depth + 1) %}{% endif %}
```

Comment:
This makes me smile.

Can you open a ticket against Weaver to provide this as a built-in filter we can use on fields?

E.g. something like:

``` 
{% for field, depth in fields | sort_by_body_field_depth %}
```

Reply:
great point! https://github.com/open-telemetry/weaver/issues/410